nautobot/ipam/models.py

```diff
@@ -650,6 +660,16 @@ def get_absolute_url(self):
     def get_duplicates(self):
         return IPAddress.objects.filter(vrf=self.vrf, host=self.host).exclude(pk=self.pk)
 
+    # TODO: The current IPAddress model has no appropriate natural key available yet.
+    #       However, by default all BaseModel subclasses now have a `natural_key` property;
+    #       but for this model, accessing the natural_key will raise an exception.
+    #       The below is a hacky way to "remove" the natural_key property from this model class for the time being.
+    class AttributeRemover:
```

Comment:
I'd prefer to keep it inline just so that it's clear this is a short-term hack - putting it into `core.utils` makes it more "official" and I'm worried that we might then have to support this antipattern long-term. :-)